src/OpenLoco/src/Windows/IndustryList.cpp

```diff
@@ -275,6 +283,50 @@ namespace OpenLoco::Ui::Windows::IndustryList
             return rhsVar < lhsVar;
         }
 
+        static uint32_t getProductionLastMonth(const OpenLoco::Industry& industry, StringId* unitType = nullptr)
+        {
+            auto industryObj = ObjectManager::get<IndustryObject>(industry.objectId);
+            uint32_t cargoProduction = std::numeric_limits<uint32_t>::max();
+
+            if (unitType)
```

Comment:
you check unitType multiple times - can we instead check it once?

Reply:
It is a nullptr when called from the comparison function.

Reply:
ah I didn't even realise, sorry about that. personally I think defaulting arguments are a bad practice, i'd prefer just passing in nullptr at the comparison part

Reply:
I've changed it to a `std::pair`, assuming that the cargo type lookup is cheap enough it won't affect lower-end systems.

Reply:
I think if the cargo type lookup is slow, we should fix that first haha, hopefully its not slow at the moment